Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:14-alpine

```

Comment:
Better to use a fixed version here instead of latest. (currently or 14.18.0 or 16.10.0)

Reply:
Version fixed
6348b91338126d857fa44ff466646d2a6e09c839